cmd/empire/main.go

```diff
@@ -98,6 +110,30 @@ var Commands = []cli.Command{
 				EnvVar: "EMPIRE_SCHEDULER",
 			},
 			cli.StringFlag{
+				Name:   FlagServerAuth,
+				Value:  "",
+				Usage:  "The authentication backend to use to authenticate requests to the API. Can be `fake`, `github`, or `saml`.",
```

Comment:
Previously, we've defaulted to `fake`, which was determined by the absence of `EMPIRE_GITHUB_CLIENT_ID`.

For backwards compatibility, I went with leaving this blank by default. If it's blank, then it'll use the existing behavior and look at `EMPIRE_GITHUB_CLIENT_ID`. We could break backwards compatibility here and error out if it's not explicitly provided, which I'd be ok with.